terra-jupyter-aou/Dockerfile

```
@@ -193,11 +193,81 @@ RUN R -e 'BiocManager::install(c("GENESIS"))'
 ENV R_LIBS /usr/local/lib/R/site-library:/opt/conda/lib/R/library
 RUN conda install -c bioconda r-saige
 
+# Install hdf5 library and h5py
+RUN apt-get update && \
+  apt install -yq --no-install-recommends \
+        libhdf5-serial-dev \
+        cmake
```

We usually include cleanup as part of `apt-get` usage to reduce the size of the resulting image.
```suggestion
RUN apt-get update && \
  apt install -yq --no-install-recommends \
        libhdf5-serial-dev \
        cmake
        && apt-get clean \
        && rm -rf /var/lib/apt/lists/*
```